notifier.go

```diff
@@ -302,6 +303,8 @@ func (n *Notifier) sendNotice(notice *Notice) (string, error) {
 		return "", errAccountRateLimited
 	case http.StatusRequestEntityTooLarge:
 		return "", errNoticeTooBig
+	case http.StatusBadRequest:
+		return "", badRequest
```

Comment:
`errBadRequest`

It probably also makes sense to parse the returned JSON and return an error from it.